package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@hmcts/one-per-page",
   "description": "One question per page apps made easy",
   "homepage": "https://github.com/hmcts/one-per-page#readme",
-  "version": "3.4.3",
+  "version": "4.0.0",
```

Comment:
I don't think this is the correct version? Is it a breaking change?

Reply:
don't think it would break, but does move baseUrl to cookie.domain setting, do you think that can be a minor?

Reply:
Well I think you only make it major if it's not backwards compatible so if these changes won't affect our apps then I think it can be a minor